python/test/unit/runtime/test_i64_print.py

```diff
@@ -0,0 +1,21 @@
+import triton
+import triton.language as tl
+import torch
+
+
+def test_i64_printf(capfd):
+
+    @triton.jit
+    def ndscore_kernel(ptr):
+        value = tl.load(ptr)
+        print("value in kernel", value)
+        tl.store(ptr, value + 1)
+
+    ptr = torch.tensor(42, dtype=torch.int64).cuda()
+    print("value before kernel", ptr.item())
+    ndscore_kernel[(1, )](ptr)
+    print("value after kernel", ptr.item())
+    captured = capfd.readouterr()
+    assert "value in kernel: 42" in captured.out
+    assert "value before kernel 42" in captured.out
+    assert "value after kernel 43" in captured.out
```

Comment:
Why is this not already covered by existing `test_print`? Can we just add this to the same file or extend the existing test.

Reply:
The issue (4060) only repros when printing scalar(Tensor.dim() == 0) i64 values. I have removed this test and added device_print_scalar to test_print